docker/final.Dockerfile

```diff
@@ -19,8 +24,8 @@ RUN apt-get -q update \
      python3-minimal \
      python3-git \
      python3-semantic-version \
-     python3-websocket \
- && apt-get clean \
+     python3-websocket
+RUN apt-get clean \
  && rm -rf /var/lib/apt/lists/*
```

Comment:
The purpose of this step is to remove cached files from `apt-get` from the layer. If you add a separate `RUN` instructions the files are not present in that layer anymore, but will be in the previous layer. This increases the size of the image (probably inconsequential in the builder image, but not good for this final image).

Reply:
Ah, true. I'll drop out that commit.